pmm-app/tests/helper/grafana_helper.js

```diff
@@ -1,26 +1,46 @@
 const Helper = codecept_helper;
 const assert = require('assert');
-const {saveVideo} = require('playwright-video');
+const { saveVideo } = require('playwright-video');
 const fs = require('fs');
 
 class Grafana extends Helper {
-
   constructor(config) {
     super(config);
-    this.resultFilesFolder = global.output_dir + "/";
+    this.resultFilesFolder = global.output_dir + '/';
   }
 
   async Authorize(username = 'admin', password = 'admin') {
-    const {browserContext} = this.helpers.Playwright;
+    const { browserContext } = this.helpers.Playwright;
     const basicAuthEncoded = await this.getAuth(username, password);
-    await browserContext.setExtraHTTPHeaders({Authorization: `Basic ${basicAuthEncoded}`});
-    return browserContext
+    await browserContext.setExtraHTTPHeaders({ Authorization: `Basic ${basicAuthEncoded}` });
+    return browserContext;
   }
 
   async getAuth(username = 'admin', password = 'admin') {
-    return Buffer.from(
-        `${this.config.username || username}:${this.config.password || password}`
-    ).toString('base64');
+    return Buffer.from(`${this.config.username || username}:${this.config.password || password}`).toString(
+      'base64'
+    );
+  }
+
+  async _before(test) {
+    const allure = codeceptjs.container.plugins('allure');
+
+    switch (true) {
+      case test.title.includes('[blocker]'):
+        allure.severity('blocker');
+        break;
+      case test.title.includes('[critical]'):
+        allure.severity('critical');
+        break;
+      case test.title.includes('[minor]'):
+        allure.severity('minor');
+        break;
+      case test.title.includes('[trivial]'):
```

Comment:
I don't see we need more than four levels, trivial is probably one too many.

Reply:
Those are all allure severities, so i thought they all should be added. Should i remove the `trivial` ?

Reply:
I think I agree with Alex T, we don't need Trivial

Reply:
I would suggest so!